### ISSUE TITLE
[BUG] Could not pass event BlockPhysicsEvent

### ISSUE THREAD

**Issue by Krosta8** [FabioZumbi12/RedProtect#609]
**Describe the bug**
With every block interact, block placed/destroyed, falling sand - error in console.
The problem is in and out of a claim.

**To Reproduce**
1. Run server with RedProtect
2. Try to interact with a block, or destroy/build a block
3. Error in console.

**Screenshots and LOGs**
Falling sand: https://gist.github.com/Krosta8/1db022fe92018f97193984818e87e3f5
Block placed: https://gist.github.com/Krosta8/b7b76f5aa83650f8bddef0e0745f5778
Chest opened: https://gist.github.com/Krosta8/8ccc3cec52f57a45467124267daefabd

**Server and plugins versions (please complete the following information):**
 - Server: Mohist version a96eaf0 (MC: 1.12.2) (Implementing API version 1.12.2-R0.1-SNAPSHOT, Forge version 14.23.5.2847)
 - Plugin Version: RedProtect-7.6.6-b307-Universal.jar
 - Java Version: Java 1.8

**Additional context**
I tried it with/without creative and with/without OP

Sorry for so many bugs report, just wanted to let you know.

**Comment by FabioZumbi12** [comment]
Thanks for detailed info. Fixed on latest dev build! (need test)

**Comment by Krosta8** [comment]
No problem :) 
I tried to test it (quickly). It seems to be without any error, but I just tried to put chest inside the region and I was able to open it, but the claim was not mine and with "chest: false"
I made server region, check that chest is set to false and I am still able to open the chest. 
I checked it also with LuckPerms and it was checking "redprotect.flag.bypass.allow-mod - undefined" when I was opening the chest. Strange :D
